src/main/java/Chris.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Scanner;
+public class Chris {
+    public static void main(String[] args) {
+        String logo = "      ___           ___           ___                       ___     \n" +
+                "     /\\  \\         /\\__\\         /\\  \\          ___        /\\  \\    \n" +
+                "    /::\\  \\       /:/  /        /::\\  \\        /\\  \\      /::\\  \\   \n" +
+                "   /:/\\:\\  \\     /:/__/        /:/\\:\\  \\       \\:\\  \\    /:/\\ \\  \\  \n" +
+                "  /:/  \\:\\  \\   /::\\  \\ ___   /::\\~\\:\\  \\      /::\\__\\  _\\:\\~\\ \\  \\ \n" +
+                " /:/__/ \\:\\__\\ /:/\\:\\  /\\__\\ /:/\\:\\ \\:\\__\\  __/:/\\/__/ /\\ \\:\\ \\ \\__\\\n" +
+                " \\:\\  \\  \\/__/ \\/__\\:\\/:/  / \\/_|::\\/:/  / /\\/:/  /    \\:\\ \\:\\ \\/__/\n" +
+                "  \\:\\  \\            \\::/  /     |:|::/  /  \\::/__/      \\:\\ \\:\\__\\  \n" +
+                "   \\:\\  \\           /:/  /      |:|\\/__/    \\:\\__\\       \\:\\/:/  /  \n" +
+                "    \\:\\__\\         /:/  /       |:|  |       \\/__/        \\::/  /   \n" +
+                "     \\/__/         \\/__/         \\|__|                     \\/__/    ";
+        System.out.println("Hello from\n" + logo );
+
+        Scanner sc = new Scanner(System.in);
+        String command = "";
+        TaskLists listCommands = new TaskLists();
+        System.out.println("____________________________________________________________\n" +
```

Comment:
can consider having different member functions under the main class to declutter the main function.